src/samos/ui/main_page.py

```diff
@@ -1298,6 +1329,32 @@ def find_stars(self):
             obj.add_callback('edited', self.edit_cb)
             self.slit_tab_view.add_slit_obj(region, obj.tag, self.fits_image)
 
+    @check_enabled
+    def show_GSP00(self):
+        # Show the position of the GSP00 on the image
+        radius_pix = 15
+        reg_GSP00 = CirclePixelRegion(center=PixCoord(self.gs_x0.get(), self.gs_y0.get()), radius=radius_pix)
+        obj = r2g(reg_GSP00)
+        obj.color = "blue"
+        obj.linewidth = 3
+        self.tag_GSP00 = '@check_GSP00_'+str(time.time())  #change the tag each time the circle is created
+        self.canvas.add(obj, tag=self.tag_GSP00)
```

Comment:
What will happen if the GSP00 is already shown? Will it draw it again? If so, why do we want to do that? Also, since this function will always replace `self.tag_GSP00` with the new value, it seems like if you click the button twice, you'll have two circles on top of each other, and the program will only be able to hide the top one, because the tag of the bottom one will no longer be known.